new/rfc000-rfc-process.md

```diff
@@ -0,0 +1,160 @@
+---
+RFC: 000
+Author: Chris McClimans <chris@hippiehacker.org>
+Status: Draft
+Type: Process
+---
+
+# ii RFC Process and Guidelines
+
+A ii RFC is a design document describing improvements to ii. We as a
+community use RFCs to document, discuss, and plan improvements to ii and
+other ii ecosystem tools. You can always find the list of accepted RFCs at
+https://github.com/instantinfrastructure/ii-rfc.
+
+## RFC Types
+
+* A **Standards Track** RFC describes a new feature or improvement for ii or
+the ii ecosystem.
+* An **Informational** RFC describes a standard or guideline in the ii
+community. It is not intended to be a binding requirement.
+* A **Process** RFC describes a change to development process of ii or
+related projects.
+
+## Roles
+
+* The **Decider** has final say to accept or reject an RFC. The top-level
+decider is [Chris McClimans](mailto:chris@hippiehacker.org). The decider can delegate their
```

Comment:
what email address should I use... @hippiehacker.org @vulk.co @instantinfrastructure.org ?